### DISCUSSION TITLE
Replace Docker image "docker/compose"

### DISCUSSION THREAD

**Discussion by mueller-ma** [renovatebot/renovate#30454] [Suggest an Idea]
### Tell us more.

The image docker/compose has been integrated in the official docker image. See https://hub.docker.com/r/docker/compose:

> This image contains Compose v1, which has reached end of life (EOL) and is no longer supported. This image’s tags are frozen and are not updated.
> If you want to use Compose directly from a container you can use the [docker official image](https://hub.docker.com/_/docker):

This image could be added to https://docs.renovatebot.com/presets-replacements/#replacementsall

**Comment by viceice** [comment]
feel free to open a PR

**Reply by mueller-ma** [reply]
- https://github.com/renovatebot/renovate/pull/30469
